src/BPGates.jl

```diff
@@ -319,8 +325,11 @@ julia> bellmeasure!(BellState([0,1,1,1]), BellMeasure(2,1))
 ```
 """
 function bellmeasure!(state::BellState, op::BellMeasure) # TODO document which index corresponds to which measurement
+    # does this apply both coincidence AND anti-coincidence measurements?
+    # i assume that it's hard coded into measure_tuple
```

Comment:
Yes, all three types of measurements that result in preserving the A state are included here.